### PR TITLE
Fix typo in translation docs

### PR DESCRIPTION
**Pull request by ShivraniAJ**
---
This change is marked as an *internal change* (Task), so will not be included in the changelog.

### CODE REVIEW

**Comment by ShivraniAJ** [comment]
Screenshot (30)
While going through docs/translating.md found one typo error in the spelling of in-deepth, so I have fixed this error.

File: docs/translating.md [hunk 1]
```diff
@@ -62,4 +62,6 @@ You can use inside the translation field "Review needed" checkbox. It will be sh
 
 ### Further reading
 
-The official Weblate doc provides some more in-deepth explanation on how to do translations and talks about do and don'ts. You can find it at: https://docs.weblate.org/en/latest/user/translating.html
+The official Weblate doc provides some more in-depth explanation on how to do translations and talks about do and don'ts. You can find it at: https://docs.weblate.org/en/latest/user/translating.html
+
+
```
Comment (turt2live) [inline docs/translating.md:67]: these lines appear to be excess for this change

**Review by turt2live — CHANGES_REQUESTED** [review]

File: docs/translating.md (hunk 1 above)
Comment (ShivraniAJ) [reply docs/translating.md:67]: > these lines appear to be excess for this change

I will do the required change.

**Review by turt2live — APPROVED** [review]
Thanks!

**Pull request merged**